examples/with-linaria/next.config.js

```diff
@@ -0,0 +1,19 @@
+const withCSS = require('@zeit/next-css')
```

Comment:
I don't think `withCSS` is needed here 🤔

Reply:
I see, looks like you're relying on it for css extraction

https://github.com/callstack/linaria/blob/master/docs/BUNDLERS_INTEGRATION.md#webpack

Reply:
I tried it, but I ran into a production problem: https://github.com/callstack/linaria/issues/318#issuecomment-466633036

The server does not see the file with styles and returns 404